### PR TITLE
+ addenda

### PR DESCRIPTION
**Pull request by romainfrancois**
I've added some addendums about things we changed thanks to the community feedback. I'm not sure why this does not work, I just added 

```
<div class="addendum">
  Some text about a function, e.g. `group_trim()`
</div>
```

and the `addendum` classes defined in the document. For some reason, the text is included verbatim.

### CODE REVIEW

File: content/articles/2018-12-dplyr-0-8-0-release-candidate.Rmarkdown [hunk 1]
```diff
@@ -15,6 +15,17 @@ photo:
   author: Pau Casals
 ---
 
+<style>
+.addendum {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  padding: 10px 10px 10px 50px;
```
Comment (hadley) [inline content/articles/2018-12-dplyr-0-8-0-release-candidate.Rmarkdown]: I'd make the padding a constant `padding: 10px`, give the box a background (pale grey), and make the border a bit thicker.

**Comment by hadley** [comment]
Otherwise, looks great!

File: content/articles/2018-12-dplyr-0-8-0-release-candidate.Rmarkdown (hunk 1 above)
Comment (romainfrancois) [reply content/articles/2018-12-dplyr-0-8-0-release-candidate.Rmarkdown]: I used rgb(248, 248, 248) to match the bg of functions

**Comment by batpigandme** [comment]
> I've not been able to use fenced divs, I guess it's a Pandoc thing and Hugo does not use that. Instead of that I ended up using block quotes and locally tweak their style.

Yeah, because we use Rmarkdown with Hugo it gets rendered by BlackFriday, which is different somehow.

As for addendum in the plural, I'd go with addenda. If nothing else, it's what the Oxford Style Manual says, and I pretty much follow that.
image

File: content/articles/2018-12-dplyr-0-8-0-release-candidate.Rmarkdown [hunk 2]
```diff
@@ -31,6 +43,13 @@ issue <- function(n, pkg = "dplyr", org = "tidyverse") {
 }
 ```
 
+> This post, published in early december and promoted on twitter
```
Comment (batpigandme) [inline content/articles/2018-12-dplyr-0-8-0-release-candidate.Rmarkdown]: Both december and twitter get first-letter uppercase

File: content/articles/2018-12-dplyr-0-8-0-release-candidate.Rmarkdown [hunk 3]
```diff
@@ -31,6 +43,13 @@ issue <- function(n, pkg = "dplyr", org = "tidyverse") {
 }
 ```
 
+> This post, published in early december and promoted on twitter 
+> generated valuable discussions that led us to reconsider some
+> design choices for `dplyr` 0.8.0
+
+> We've left the original post unchanged and added addendums when
```
Comment (batpigandme) [inline content/articles/2018-12-dplyr-0-8-0-release-candidate.Rmarkdown]: addendums ⇨ addenda

File: content/articles/2018-12-dplyr-0-8-0-release-candidate.Rmarkdown [hunk 4]
```diff
@@ -166,6 +185,10 @@ df %>%
   filter(y < 5, .preserve = FALSE)
 ```
 
+>  As opposed to what is described above, feedback from this post led us
+>  to change the default value of `.preserve` to `FALSE` and update the
```
Comment (batpigandme) [inline content/articles/2018-12-dplyr-0-8-0-release-candidate.Rmarkdown]: comma after False

File: content/articles/2018-12-dplyr-0-8-0-release-candidate.Rmarkdown [hunk 5]
```diff
@@ -262,6 +296,9 @@ mtcars %>%
 The lambda function must return a data frame. `r link("group_map")` row binds the data 
 frames, recycles the grouping columns and structures the result as a grouped tibble. 
 
+>  `group_walk()` can be used when iterating on the groups is only desired for side effects.
```
Comment (batpigandme) [inline content/articles/2018-12-dplyr-0-8-0-release-candidate.Rmarkdown]: I think if you're referring to the same function here, you can use "It" in lieu of repeating the function name.

**Comment by romainfrancois** [comment]
Thanks for the tweaks, 🎯 as always. Should be good now.

**Comment by batpigandme** [comment]
@hadley you cool with the state of the boxes?

**Comment by hadley** [comment]
I don't love the asymmetric padding (probably because of a margin on something inside the blockquote), but I don't care too much. But I do think the text should not be in italics, and the font size should be the same as the rest of the text.

**Comment by romainfrancois** [comment]
image

**Comment by romainfrancois** [comment]
I think it's better now. I'll tweet it again, with some screenshots to let people know the addenda are there, thanks to the comments we've had so far.

**Comment by batpigandme** [comment]
OK, I was just waiting for @hadley to confirm the status of the boxes

**Pull request merged**